### PR TITLE
400 edit msg template latest

### PR DESCRIPTION
**Pull request by gborgonia-ir**
Ticket ID:
[Create Edit Message Template screen](https://github.com/nerubia/kh360-react/issues/400)

BE PR: https://github.com/nerubia/kh360-nodejs/pull/423

### CODE REVIEW

File: src/features/admin/email-templates/[id]/edit/edit-email-template-form.tsx [hunk 1]
```diff
@@ -0,0 +1,282 @@
...
+    is_default: false,
+    subject: "",
+    content: "",
+  })
+
+  const [validationErrors, setValidationErrors] = useState<Partial<EmailTemplateFormData>>({})
+  const [showDialog, setShowDialog] = useState<boolean>(false)
+  const [showDefaultDialog, setShowDefaultDialog] = useState<boolean>(false)
+  const [isDefault, setIsDefault] = useState<boolean>(false)
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (jatnerubia) [inline src/features/admin/email-templates/[id]/edit/edit-email-template-form.tsx:50]: avoid using `any`

File: src/features/admin/email-templates/[id]/edit/edit-email-template-form.tsx
Comment (jatnerubia) [inline src/features/admin/email-templates/[id]/edit/edit-email-template-form.tsx:1]: can you reuse the email template form?

File: src/features/admin/email-templates/email-template-form/email-template-form.tsx [hunk 2]
```diff
@@ -46,8 +47,9 @@ export const EmailTemplateForm = () => {
   const [validationErrors, setValidationErrors] = useState<Partial<EmailTemplateFormData>>({})
   const [showDialog, setShowDialog] = useState<boolean>(false)
   const [showDefaultDialog, setShowDefaultDialog] = useState<boolean>(false)
-  const [updateDefault, setUpdateDefault] = useState<boolean>(false)
   const [template_type, setTemplateType] = useState<string>(searchParams.get("template_type") ?? "")
+  // eslint-disable-next-line @typescript-eslint/no-explicit-any
```
Comment (jatnerubia) [inline src/features/admin/email-templates/email-template-form/email-template-form.tsx:51]: avoid using `any`

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

File: src/pages/admin/email-templates/[id]/edit/edit-email-template.tsx [hunk 3]
```diff
@@ -0,0 +1,12 @@
+import { useTitle } from "../../../../../hooks/useTitle"
+import { EmailTemplateForm } from "../../../../../features/admin/email-templates/email-template-form/email-template-form"
+
+export default function EditEvaluation() {
```
Comment (jatnerubia) [inline src/pages/admin/email-templates/[id]/edit/edit-email-template.tsx:4]: update the component name

**Review by jatnerubia — CHANGES_REQUESTED** [review]
just a few changes ^^

also please update the name [here](https://github.com/nerubia/kh360-react/blob/beta/src/pages/admin/email-templates/email-templates.tsx)

**Review by jatnerubia — APPROVED** [review]
Cool

**Review by emmaacha — APPROVED** [review]

**Pull request merged**